player/js/module_worker.js

```diff
@@ -56,22 +56,86 @@ var lottie = (function () {
   return lottiejs;
 }({}));
 
-var animations = [];
```

Comment:
I think that we don't really need this, cause we will start one animation per worker so every animation will have its own thread